Tests/AuthenticationTests/LoginSessionConfigurationTests.swift

```diff
@@ -2,44 +2,63 @@
 import XCTest
 
 final class LoginSessionConfigurationTests: XCTestCase {
-
+    
     var sut: LoginSessionConfiguration!
+    
+    override func setUp() {
+        super.setUp()
+        sut = LoginSessionConfiguration(authorizationEndpoint: URL(string: "https://www.google.com")!,
+                                        tokenEndpoint: URL(string: "https://www.google.com/token")!,
+                                        responseType: .code,
+                                        scopes: [.email],
+                                        clientID: "1234",
+                                        prefersEphemeralWebSession: true,
+                                        redirectURI: "https://www.google.com/redirect",
+                                        nonce: "1234",
+                                        viewThroughRate: "1",
+                                        locale: .en)
+    }
+    
+    override func tearDown() {
+        sut = nil
         
-        override func setUp() {
-            super.setUp()
-            let url = URL(string: "https://www.google.com")!
-
-            sut = .init(LoginSessionConfiguration(authorizationEndpoint: url,
-                                                  responseType: .code,
-                                                  scopes: [.email],
-                                                  clientID: "1234",
-                                                  prefersEphemeralWebSession: true,
-                                                  redirectURI: "https://www.google.com",
-                                                  nonce: "1234",
-                                                  viewThroughRate: "1",
-                                                  locale: .en))
-        }
-        
-        override func tearDown() {
-            sut = nil
-            
-            super.tearDown()
-        }
+        super.tearDown()
+    }
 }
 
 extension LoginSessionConfigurationTests {
     func testSUTHasAuthorizationURL() {
         XCTAssertEqual(sut.authorizationEndpoint, URL(string: "https://www.google.com"))
     }
     
+    func testSUTHasTokenURL() {
+        XCTAssertEqual(sut.tokenEndpoint, URL(string: "https://www.google.com/token"))
+    }
+    
     func testResponseType() {
         XCTAssertEqual(sut.responseType, .code)
     }
     
+    func testDefaultResponseType() {
+        sut = LoginSessionConfiguration(authorizationEndpoint: URL(string: "https://www.google.com")!,
+                                        tokenEndpoint: URL(string: "https://www.google.com/token")!,
+                                        clientID: "1234",
+                                        redirectURI: "https://www.google.com/redirect")
+        XCTAssertEqual(sut.responseType, .code)
```

Comment:
do these all need to be separate tests or worth combining them into one test with a bunch of assertions?